crates/iota-node/src/admin.rs

```diff
@@ -204,6 +204,10 @@ async fn enable_tracing(
             response.push(format!("filter will be reset after {:?}", duration));
             (StatusCode::OK, response.join("\n"))
         }
+        Err(TelemetryError::TracingDisabled) => {
+            response.push("tracing is not supported as it was disabled in the node's TelemetryConfig\nignoring request".into());
+            (StatusCode::FORBIDDEN, response.join("\n"))
```

Comment:
```suggestion
            (StatusCode::NOT_IMPLEMENTED, response.join("\n"))
```

I would rather use the "not implemented" here, because forbidden is more due to policy or permission issues.
"service unavailable" sounds better, but unfortunately that is more in terms of something is currently broken.
"not implemented" is the best fit I guess.

Reply:
yeah, I was considering this one too, sounds right